cinema/models.py

```diff
@@ -95,19 +95,20 @@ def clean(self):
             if not (1 <= ticket_attr_value <= count_attrs):
                 raise ValidationError(
                     {
-                        ticket_attr_name: f"{ticket_attr_name} "
-                        f"number must be in available range: "
-                        f"(1, {cinema_hall_attr_name}): "
-                        f"(1, {count_attrs})"
+                        ticket_attr_name:
+                            f"{ticket_attr_name} "
+                            f"number must be in available range: "
+                            f"(1, {cinema_hall_attr_name}): "
+                            f"(1, {count_attrs})"
```

Comment:
The error message in the `ValidationError` could be formatted more clearly. Consider simplifying the message to directly state the range, such as: `f'{ticket_attr_name} number must be between 1 and {count_attrs}'`. This makes the message more concise and easier to understand.